content/99.citation-tags.md

```diff
@@ -3,4 +3,4 @@
 [@tag:garson-interpreting]: https://dl.acm.org/citation.cfm?id=129452
 [@tag:krogh-weight-decay]: http://dl.acm.org/citation.cfm?id=2986916.2987033
 [@tag:predicting-pneumonia-mortality]: doi:10.1016/S0933-3657(96)00367-3
-[@tag:Yosinski2014]: https://papers.nips.cc/paper/5347-how-transferable-are-features-in-deep-neural-networks
+[@tag:Yosinski2014]: https://dl.acm.org/doi/abs/10.5555/2969033.2969197
```

Comment:
Can we just use a regular DOI tag?

Reply:
NeurIPS doesn't assign DOIs as far as I know.  Despite what is implied by the ACM URL, `10.5555/2969033.2969197` isn't a valid DOI.

The original URL https://papers.nips.cc/paper/5347-how-transferable-are-features-in-deep-neural-networks was redirecting to https://proceedings.neurips.cc/paper/1987/file/1679091c5a880faf6fb5e6087eb1b2dc-Paper.pdf for some reason, so I swapped it for the ACM URL.  That seems to work.
```
$ manubot cite --render https://dl.acm.org/doi/abs/10.5555/2969033.2969197
1. How transferable are features in deep neural networks?
Jason Yosinski, Jeff Clune, Yoshua Bengio, Hod Lipson
Proceedings of the 27th International Conference on Neural Information Processing Systems - Volume 2 (2014-12-08) https://dl.acm.org/doi/abs/10.5555/2969033.2969197
```

Reply:
That's wild. Good change then!